### PR TITLE
Add sound when jumping on the trampoline

### PR DESCRIPTION
**Pull request by NullLikesNothing**

### CODE REVIEW

File: lua/entities/cfc_trampoline/cl_init.lua [hunk 1]
```diff
@@ -3,3 +3,11 @@ include( "shared.lua" )
 function ENT:Draw()
     self:DrawModel()
 end
+
+local BOUNCE_SFX_VOLUME = CreateClientConVar( "cfc_trampoline_volume", 1, true, false, "Volume for the trampoline", 0, 1 )
+
+net.Receive( "CFC_BouncyCircle_PlayBounceSound", function()
+    local ent = net.ReadEntity()
+
+    ent:EmitSound( "cfc/cfc_trampoline/cfc_trampoline_bounce.ogg", 75, 100, BOUNCE_SFX_VOLUME:GetFloat() )
```
Comment (brandonsturgeon) [inline lua/entities/cfc_trampoline/cl_init.lua:12]: We may need an `IsValid` check here. I think the item could be in your PVS but not your PAS? Not 100% sure.
Let's merge it without and then see if clients error.

Also, a specific [CHAN](https://gmodwiki.com/Enums/CHAN) might be good here but I don't know which one. It's probably fine without.

File: lua/entities/cfc_trampoline/init.lua [hunk 2]
```diff
@@ -4,6 +4,7 @@ AddCSLuaFile( "cl_init.lua" )
 AddCSLuaFile( "shared.lua" )
 
 include( "shared.lua" )
+util.AddNetworkString( "CFC_BouncyCircle_PlayBounceSound" )
```
Comment (brandonsturgeon) [inline lua/entities/cfc_trampoline/init.lua:7]: This should only be run on server. If you don't have a server file, just wrap an `if SERVER` around it

File: lua/entities/cfc_trampoline/init.lua [hunk 3]
```diff
@@ -159,7 +160,9 @@ function ENT:StartTouch( ent )
     local upSpeed = entVel:Dot( trampolineDown )
     if upSpeed < MIN_SPEED:GetFloat() then return end
 
-    self:EmitSound( "cfc_trampoline_bounce.wav" )
+    net.Start( "CFC_BouncyCircle_PlayBounceSound" )
+    net.WriteEntity( self )
+    net.SendPAS( self:GetPos() )
```
Comment (brandonsturgeon) [inline lua/entities/cfc_trampoline/init.lua:165]: This probably fits better inside the `Bounce` function itself

**Review by brandonsturgeon — APPROVED** [review]
Looks good! Excited to try it. Just a few code structure comments 👍

File: lua/entities/cfc_trampoline/init.lua (hunk 2 above)
Comment (NullLikesNothing) [reply lua/entities/cfc_trampoline/init.lua:7]: That is in a server only file

**Comment by brandonsturgeon** [comment]
Looks good. Be sure to run the workshop deploy action

**Pull request merged**